static/_redirects

```diff
@@ -369,3 +369,11 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes.md https://github.com/ava-labs/avalanchego/releases 301
+/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
```

Comment:
What is the reason for this and following two URLs?

Reply:
If external sites are referencing https://docs.avax.network/apis/avalanchego/avalanchego-release-notes we'd want to redirect them to the official notes in GH

Reply:
> If external sites are referencing https://docs.avax.network/apis/avalanchego/avalanchego-release-notes we'd want to redirect them to the official notes in GH

But the first part is /avalanchego-release-notes/, not /apis/avalanchego/avalanchego-release-notes